tasks/spec.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- name: initialize function
+  include_tasks: includes/init.yaml
+
```

Comment:
need to add `when: yang_fetch_schema|bool == false` condition to ensure init.yaml task is not included twice (since spec task include fetch task when `yang_fetch_schema` is set)